### PR TITLE
ToggleSwitch RTL

### PR DESCRIPTION
**Pull request by alisey**

### CODE REVIEW

File: packages/wix-ui-core/src/components/StylableToggleSwitch/ToggleSwitch.spec.tsx [hunk 1]
```diff
@@ -81,6 +81,11 @@ describe('ToggleSwitch', () => {
       const driver = createDriver(<ToggleSwitch tabIndex={7} />);
       expect(driver.getTabIndex()).toBe(7);
     });
+
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/components/StylableToggleSwitch/ToggleSwitch.spec.tsx]: I would add a test for ltr by default

File: packages/wix-ui-core/src/components/StylableToggleSwitch/ToggleSwitch.tsx [hunk 2]
```diff
@@ -50,7 +51,9 @@ export class ToggleSwitch extends React.PureComponent<ToggleSwitchProps, ToggleS
     /** Icon inside of the knob when checked */
     checkedIcon: propTypes.node,
     /** Icon inside of the knob when unchecked */
-    uncheckedIcon: propTypes.node
+    uncheckedIcon: propTypes.node,
+    /** Direction */
+    dir: propTypes.oneOf(['ltr', 'rtl'])
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/components/StylableToggleSwitch/ToggleSwitch.tsx]: what about adding dafaultProps.dir = 'ltr' ?

**Comment by alisey** [comment]
LTR by default, added test.

**Comment by shlomitc** [comment]
are we sure the using `ltr` is needed? shouldn't it just be the default behavior and only when you need `rtl` pass it as a prop (or use it from the `body`)?

**Pull request closed without merging**